server/resp/src/main/java/org/infinispan/server/resp/commands/json/JSONGET.java

```diff
@@ -0,0 +1,86 @@
+package org.infinispan.server.resp.commands.json;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+import org.infinispan.server.resp.json.EmbeddedJsonCache;
+import org.infinispan.server.resp.serialization.ResponseWriter;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * JSON.GET
+ *
+ * @see <a href="https://redis.io/commands/json.get/">JSON.GET</a>
+ * @since 15.2
+ */
+public class JSONGET extends RespCommand implements Resp3Command {
+
+   public JSONGET() {
+      super("JSON.GET", -1, 1, 1, 1);
+   }
+
+   @Override
+   public long aclMask() {
+      return 0;
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx,
+         List<byte[]> arguments) {
+      byte[] key = arguments.get(0);
+      Args args;
+      args = parseArgs(arguments);
+      if (args == null) {
+         handler.writer().wrongArgumentNumber(this);
```

Comment:
Not sure if the if this piece of code is reachable:

```
127.0.0.1:6379> json.get
(error) ERR index out of range

Reply:
> Not sure if the if this piece of code is reachable:
> 
> ```
> 127.0.0.1:6379> json.get
> (error) ERR index out of range
> ```

That part is ok, it returns null if a format argument is passed without a value.
But there was an error in the arity of the command, they should count also the command name. Fixed thanks